package.json

```diff
@@ -36,6 +36,7 @@
     "@types/redux-mock-store": "0.0.6",
     "@types/sinon": "^1.16.32",
     "chai": "^3.5.0",
+    "cheerio": "^0.22.0",
```

Comment:
I see you are using this in the unit tests, but I would need more context to understand why we need it can you please ellaborate an answer in this comment thread? Thanks